src/components/layout/header/Search.tsx

```diff
@@ -102,11 +102,11 @@ export function Search() {
     },
     {
       icon: <IconMovie />,
-      disabled: !(isOverseerrEnabled === true && overseerrApp !== undefined),
+      disabled: !(isOverseerrEnabled && overseerrApp),
       label: t('searchEngines.overseerr.name'),
       value: 'overseerr',
       description: t('searchEngines.overseerr.description'),
-      url: `${overseerrApp?.url}search?query=`,
+      url: `${overseerrApp?.url.replace(new RegExp('/' + "+$"),'')}/search?query=`,
```

Comment:
What is up with this?
Consider to document using a comment.

Reply:
Replace / with " " I think. Why use a regex for this?
If you just need to strip a trailing slash

Reply:
Sometimes the user doesn't put a trailing '/' at the end of their url but sometimes they do, so here I remove any trailing / if there are any and then  slap the "/search?query=" at the end

Reply:
nope, regex targets the last with +$